admin/database/database.go

```diff
@@ -115,9 +116,14 @@ type DB interface {
 	CheckUserIsAProjectMember(ctx context.Context, userID, projectID string) (bool, error)
 
 	InsertUsergroup(ctx context.Context, opts *InsertUsergroupOptions) (*Usergroup, error)
+	UpdateUsergroupName(ctx context.Context, name, groupID, orgID string) (*Usergroup, error)
+	DeleteUsergroup(ctx context.Context, groupID string) error
+	FindUsergroupByName(ctx context.Context, name, orgID string) (*Usergroup, error)
 	FindUsergroupsForUser(ctx context.Context, userID, orgID string) ([]*Usergroup, error)
 	InsertUsergroupMember(ctx context.Context, groupID, userID string) error
+	FindUsergroupMembers(ctx context.Context, groupID string) ([]*UsergroupMember, error)
 	DeleteUsergroupMember(ctx context.Context, groupID, userID string) error
+	DeleteUsergroupsMember(ctx context.Context, userID, orgID string) error
```

Comment:
Can it return the `Member` type instead of `UsergroupMember`? And maybe be renamed to `FindUsergroupMemberUsers`, `InsertUsergroupMemberUser`, etc.

Then it would align completely with the related functions for org and project membership (e.g. `FindOrganizationMemberUsers`, which returns `Member`). We should consider renaming the `type Member` to `type MemberUser` to clarify that it is a member that is a user.